integrations/vault/auths/token/src/main/java/io/helidon/integrations/vault/auths/token/TokenVaultAuth.java

```diff
@@ -77,26 +78,27 @@ public Optional<RestApi> authenticate(Config config, Vault.Builder vaultBuilder)
 
     private RestApi restApi(Vault.Builder vaultBuilder, String token) {
         String address = vaultBuilder.address()
-                .orElseThrow(() -> new VaultApiException("Address is required when using token authentication"));
+                                     .orElseThrow(() -> new VaultApiException(
+                                             "Address is required when using token authentication"));
 
         LOGGER.log(Level.INFO, "Authenticated Vault " + address + " using a token");
 
         return VaultRestApi.builder()
-                .webClientBuilder(builder -> {
-                    builder.config(vaultBuilder.config().get("webclient"))
-                            .baseUri(address + "/v1")
-                            .addHeader("X-Vault-Token", token);
-                    Optional.ofNullable(baseNamespace)
-                            .or(vaultBuilder::baseNamespace)
-                            .ifPresent(ns -> builder.addHeader("X-Vault-Namespace", ns));
-                    vaultBuilder.webClientUpdater().accept(builder);
-                })
-                .faultTolerance(vaultBuilder.ftHandler())
-                .build();
+                           .webClientBuilder(builder -> {
+                               builder.config(vaultBuilder.config().get("webclient"))
+                                      .baseUri(address + "/v1")
+                                      .header(Http.Header.create("X-Vault-Token"), token);
```

Comment:
overloaded header method

Reply:
use constant for `HeaderName`, do not create it from a constant string, do not replace it with `String` shortcut

Reply:
Updated.